tests/integration/pages/experiment_detail.py

```diff
@@ -0,0 +1,34 @@
+"""Representaion of the Experiment Detail Page."""
+
+import random
+import string
+
+from selenium.webdriver.common.by import By
+from pypom import Region
+
+from pages.base import Base
+
+class DetailPage(Base):
+
+    _edit_branches_btn_locator = (By.CSS_SELECTOR,"#branches-edit-btn")
+
+    _page_wait_locator = (
+        By.CSS_SELECTOR,
+        "#overview",
+    )
+
+    def wait_for_page_to_load(self):
+        self.wait.until(
+            lambda _: "Overview"
+            in self.find_element(*self._page_wait_locator).text
+        )
+        return self
+
+    def click_edit(self):
+        self.find_element(*self._edit_branches_btn_locator).click()
+
+        from pages.experiment_branches import BranchesPage
+
+        return DetailPage(
```

Comment:
This function seems to just return this class. Looking at the test runs, the class needing to be returned should have a function named ```input_firefox_pref_name()```.